dmapiclient/base.py

```diff
@@ -1,10 +1,7 @@
-from __future__ import absolute_import
 import logging
-
-try:
-    import urlparse
-except ImportError:
-    import urllib.parse as urlparse
+import urllib.request
```

Comment:
```suggestion
import urllib.request
```
Should we remove this? We don't seem to use `urllib.request`

Reply:
Yes, it looks like this import was added by the [fixer](https://github.com/AusDTO/dto-digitalmarketplace-apiclient/commit/b91ebae37a11de070911f2695be52b9897397ade#diff-846a0177861852a2fdc309db3dc53ce16dd85a70d73758fe22adcbcc83972ed2).